doc/conf.py

```diff
@@ -77,6 +77,7 @@ class TypeMock(type):
                  "Number": int,
                  "Tensor": list,
                  "Variable": list,
+                 "__version__": "1.3",
```

Comment:
This is done to 'trick' Strawberry Fields into letting it import the `tf` backend without TensorFlow being installed.

Reply:
might as well write that directly as a comment in file